### PR TITLE
smarter up/down behaviour for sizeref

### PR DESCRIPTION
**Pull request by nicolaskruchten**

### CODE REVIEW

**Comment by VeraZab** [comment]
💃

File: src/components/widgets/NumericInput.js [hunk 1]
```diff
@@ -80,15 +80,23 @@ export default class NumericInput extends Component {
...
   incrementValue(direction) {
-    const {defaultValue, min, step = 1} = this.props;
+    const {defaultValue, min, step = 1, stepmode = 'absolute'} = this.props;
     const {value} = this.state;
 
     let valueUpdate;
     if (isNumeric(value)) {
       if (direction === 'increase') {
-        valueUpdate = parseFloat(value) + step;
+        if (stepmode === 'absolute') {
```
Comment (dmt0) [inline src/components/widgets/NumericInput.js]: this one just screams "ternary expression" :)

Comment (nicolaskruchten) [reply src/components/widgets/NumericInput.js]: fair enough :)

**Comment by dmt0** [comment]
💃

**Pull request merged**